code/vitest.config.ts

```diff
@@ -2,6 +2,10 @@ import { coverageConfigDefaults, defineConfig } from 'vitest/config';
 
 export default defineConfig({
   test: {
+    env: {
+      NODE_ENV: 'test',
```

Comment:
what lead to this change? 👀

Reply:
Some tests erroring when running with `test-command` instead of `NODE_ENV=test test-command`